docs/content/recipes/apache.md

```diff
@@ -12,7 +12,7 @@ Usually, that includes enterprise setups using LDAP/AD on the backend and a SSO
 
 ### Alternatives
 
-If you just want authentication for your registry, and are happy maintaining users access separately, you should really consider sticking with the native [basic auth registry feature](/about/deploying#native-basic-auth).
+If you just want authentication for your registry, and are happy maintaining users access separately, you should really consider sticking with the native [basic auth registry feature](../../about/deploying#native-basic-auth).
```

Comment:
These links seem to be broken when rendering on GitHub. Wondering if it works to link to the _markdown_ file instead. I _think _ Hugo replaces those with the correct links; in that case I _think_ they also need one `../` level removed, but haven't tried;


```suggestion
If you just want authentication for your registry, and are happy maintaining users access separately, you should really consider sticking with the native [basic auth registry feature](../about/deploying.md#native-basic-auth).
```

Reply:
The pathing I used there is correct btw, at least for hugo.  The `(.*\.md)` I'd avoid personally since encoding the template isn't really a norm in hugo- in *my limited experience*.  I'm not sure I'd worry about the github linkage anyways, since nothing else in this repo has that.